challenge_opener/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
@js-gonzalez Please, add a new line at the end of the line. (See https://gist.github.com/camh-/1bebfcff1b0f814e9b191edc60d5206b)